chat-plugins/customavatar.js

```diff
@@ -39,6 +39,7 @@ exports.commands = {
 
 			var name = toId(parts[0]);
 			var image_url = parts[1];
+			if (image_url.indexOf('http://') !== 0 && image_url.indexOf('https://') !== 0) image_url = 'http://' + image_url;
```

Comment:
Can you combine that boolean into one using regex?

Ex: ``image_url.test(/https?:\/\//i)``